### PR TITLE
Sublabel scrolling

### PR DESCRIPTION
**Pull request by Chrscool8**
Hey, guys! This doesn't need to actually be merged yet, but I was curious what you think of this idea. This is still pretty much proof of concept at this point, but it works for this specific case.

Video:
https://youtu.be/AuuXQMFAQ10

Pic:

(I made the length extra small to show it off in the above media.)

I was just curious if you had any ideas (or already-made functionality that I missed!) on implementation for this sort of thing and if you think it should be pursued. 

Thanks! Keep up the great work!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ build.nx/
 *.elf
 .vscode
 builddir
+build_and_run.sh
```
Comment (natinusala) [inline .gitignore:11]: 🥖

**Comment by natinusala** [comment]
Hey, thanks for that PR!

This is a little bit awkward because @WerWolv already started working on that feature, I think it's in the fancy pants PR...?

His implementation is closer to how it looks in HOS: https://photos.app.goo.gl/WVncK13Y5wTUSvF18

And it is also generic to all Labels instead of hardcoded in ListItem like you did. Obviously it required rewriting ListItem to use Labels.

What would you think of finishing his work, or redoing your PR using the same method? There was an issue with a 10px offset compared to master, that he couldn't find the reason for.

**Comment by Chrscool8** [comment]
Oh, even better! Happy to hear it! I skimmed the other PRs but must have missed that part. 

Yeah, if I were to do it for real, I’d redo the whole code stuff to be more general, just wanted to see the look in action. :)

Thanks!

File: .gitignore (hunk 1 above)
Comment (Chrscool8) [reply .gitignore:11]: 😅

**Comment by natinusala** [comment]
@WerWolv can you maybe give info on the current state of your PR and what's left to do?

**Comment by WerWolv** [comment]
I think the only stuff that's really missing is the 10px offset shit I couldn't figure out and that there's no proper fade out animation. I wonder if this could be fixed now with deko

**Comment by Chrscool8** [comment]
Is there a trick to getting that other PR code to build? I'm getting quote warnings that I don't see a problem with and there are a couple argument errors (libnx updates since then?).

**Comment by natinusala** [comment]
Did you rebase on master first?

**Comment by Chrscool8** [comment]
I hadn't, but I tried just now and it looks like there's a lot of stuff to sift through... I'll try again in a bit!

**Comment by natinusala** [comment]
Closed as this PR is reimplemented or made irrelevant by the changes on the `yoga` branch. Scrolling is now automatically handled by all labels.

**Pull request closed without merging**